### ISSUE TITLE
.dbeaver-data-sources config file no longer working in 6.1.3

### ISSUE THREAD

**Issue by zzztimbo** [dbeaver/dbeaver#6415]
Based on the instructions found in https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Connections and https://gist.github.com/rajendarreddyj/65acf28f122d34bec8f4949c160bfd38

I wrote a Python script that will generate my db connections/credentials and write them to: `~/.dbeaver4/General/.dbeaver-data-sources-dynamic.xml`. I have to do this because my company makes us use leased credentials that expire every day in order to access our dbs. 

As of version 6.1.3, the data sources xml file I generate no longer works. I get a password authentication failed for user error. 

Going forward, I would like to still be able to generate this connections file so that I don't have to copy paste username/passwords for over a dozen connections.

**Comment by serge-rider** [comment]
Similar to #6390

In fact you still can use old xml files configuration (just delete .dbeaver folder) - but only once. Any config changes will be saved in the new format.

We will update wiki documentation shortly (#6417)

**Comment by zzztimbo** [comment]
@serge-rider what is your recommendation for my situation? i have to generate credentials every day that i want to work with my dbs. should i attempt to generate new modern format config files?

**Comment by herbinator** [comment]
Not sure if i have to create new issue. I have 2 dbeaver-data-sources.xml files. After upgrading to 6.1.4 it looks ok. Then i copy one connection to create an extra one. Closing dbeaver and opening again makes me lose all connections which used to be in the second dbeaver-data-sources-Extra.xml file. Deleting the .dbeaver file works to get everything back but off course without the newly created connection.

**Comment by serge-rider** [comment]
@zzztimbo Sorry, I missed your answer.
Please see #7248 comments for solution.
